lib/analysis_options.yaml

```diff
@@ -96,23 +108,29 @@ linter:
     library_prefixes: true # flutter
     library_private_types_in_public_api: true # flutter
     lines_longer_than_80_chars: true
+    matching_super_parameters: true
     missing_whitespace_between_adjacent_strings: true
+    no_default_cases: false
```

Comment:
Why? If you don't want to catch each statement it might better to just use a regular `if` statement and use the `else` block at that point, or in the case of return cases just define the "default" outside of the case block.

Could you give an example of a case where you want to use `default`?